sdk/examples/how_tos/advanced_transactions/claim_transaction.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will claim all claimable outputs.
+//! Rename `.env.example` to `.env` first.
+//!
+//! `cargo run --release --all-features --example claim_transaction`
+
+use iota_sdk::wallet::{account::OutputsToClaim, Result, Wallet};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    // Create the wallet
+    let wallet = Wallet::builder().finish().await?;
+
+    // Get the account we generated with `01_create_wallet`
+    let account = wallet.get_account("Alice").await?;
+
+    // May want to ensure the account is synced before sending a transaction.
+    account.sync(None).await?;
+
+    // Set the stronghold password
+    wallet
+        .set_stronghold_password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
+        .await?;
+
+    let output_ids = account
+        .get_unlockable_outputs_with_additional_unlock_conditions(OutputsToClaim::All)
+        .await?;
+    println!("Available outputs to claim:");
+    for output_id in &output_ids {
+        println!("{}", output_id);
+    }
+
+    let transaction = account.claim_outputs(output_ids).await?;
```

Comment:
`claim all claimable outputs`
If the output_ids are more than can fit in a tx, this will fail :trollface: